llvm/lib/SandboxIR/SandboxIR.cpp

```diff
@@ -3249,4 +3249,12 @@ void BasicBlock::dumpOS(raw_ostream &OS) const {
     }
   }
 }
+
+void BasicBlock::verify() const {
```

Comment:
This will need to be protected by `#ifndef NDEBUG`. Ask me how I know.

Reply:
It is in the same `#ifdef NDEBUG` block as the `dumpOS` method right above it, isn't it?